### PR TITLE
use the --debug flag on CC directly to generate pseudo names

### PR DESCRIPTION
**Pull request by erwinmombay**
since we removed the runner dependency for single pass, we werent able to generate pseudo names for debugging purposes. lets use --debug directly (CC option) to allow for this and remove the `--DEFINE='PSEUDO_NAMES=true'` indirection

### CODE REVIEW

File: build-system/compile/compile.js [hunk 1]
```diff
@@ -298,6 +295,7 @@ function compile(entryModuleFilenames, outputDir, outputFilename, options) {
       jscomp_off: [],
       define,
       hide_warnings_for: hideWarningsFor,
+      debug: !!argv.pseudo_names,
```
Comment (rsimha) [inline build-system/compile/compile.js]: For a closer mapping of `gulp dist` args to closure compiler args, why not change this to `argv.debug`?

Comment (erwinmombay) [reply build-system/compile/compile.js]: id rather not have users have to relearn a new flag

**Comment by erwinmombay** [comment]
I'd like to keep both, but add a warning to use `--debug` in the future (maybe keep it around for a month?). @rsimha @rcebulko does that sounds good?

**Comment by rsimha** [comment]
There isn't a huge audience for this flag. I'd go with whatever you and Justin prefer :)

**Comment by rcebulko** [comment]
@erwinmombay Seems like a good strat, then in a few months if you remove it no one can really complain

File: build-system/tasks/dist.js [hunk 2]
```diff
@@ -98,6 +99,13 @@ function transferSrcsToTempDir() {
  * @return {!Promise}
  */
 async function dist() {
+  if (argv.pseudo_names) {
+    log(
+      bold(yellow('--pseudo_names is deprecated, Please use --debug instead.'))
+    );
+    // Give a 3 second delay for user to see the warning.
+    await sleep(3000);
```
Comment (rcebulko) [inline build-system/tasks/dist.js]: :clap:

**Comment by mrjoro** [comment]
Is this PR ready to go in?

**Pull request closed without merging**